src/hooks/screen/useChatScreenHook.ts

```diff
@@ -73,6 +74,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
   const sendChat = async (
     message: string = randomString(20),
+    attachments: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` parameter is initialized as an empty array. If this function is expected to be called with attachments, it would be better to set the default value as `null` or `undefined`. This way, you can check if attachments were provided or not. An empty array might imply that attachments were provided but they were empty.
```diff
-    attachments: [],
+    attachments: null,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->